src/components/deposit/LotSizeSelector.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+
+import Check from '../svgs/Check'
+
+const LotSizeOption = ({ lotSize, onClick, selected }) => {
+  const handleClick = () => {
+    onClick(lotSize)
+  }
+  return (
+    <li className={classNames('lot-size-option', { selected })}
+      onClick={handleClick}>
+      <button>
+        <span className="lot-size-option-label">
+          {lotSize}&nbsp;&nbsp;฿
```

Comment:
I will confess that double-`&nbsp;` makes me twitch a little bit :laughing: Could we use `letter-spacing` instead of banking on the width of a space here?

Reply:
😆 I was afraid you'd say that - I was trying to get around adding extra `<span>`s. `letter-spacing` would affect the number spacing as well. I could add the `<span>`s and use padding or margin with `display: inline-block`.

Reply:
Oh of course lol. No I think this is survivable as-is, honestly. Was just hoping we could do something  different. Spans are probably equally bad, and require additional work.